quesma/queryparser/pancake_top_hits.go

```diff
@@ -28,25 +29,41 @@ func (p *pancakeSqlQueryGenerator) generateSimpleTopHitsQuery(topHits *pancakeMo
 		Columns:     p.aliasedExprArrayToExpr(newSelects),
 		FromClause:  model.NewTableRef(model.SingleTableNamePlaceHolder),
 		WhereClause: whereClause,
-		OrderBy:     topHitsQueryType.OrderBy,
-		Limit:       topHitsQueryType.Size,
+		OrderBy:     orderBy,
+		Limit:       size,
 	}
 
 	return resultQuery, nil
 }
 
-func (p *pancakeSqlQueryGenerator) generateTopHitsQuery(aggregation *pancakeModel,
+func (p *pancakeSqlQueryGenerator) generateTopQuery(aggregation *pancakeModel,
```

Comment:
change `topHits` name, also maybe some comment here as well

Reply:
I will fix and add comment.